tests/test_efficiency.py

```diff
@@ -16,12 +16,23 @@ def test_efficiency_subsample():
         assert True
 
 
+def simple_efficiency_uncertainty(total, sample):
```

Comment:
This is derived from Binomial Statistics, please add it to the name of the function or to the docstring.